Source/WebCore/html/shadow/SliderThumbElement.cpp

```diff
@@ -554,29 +549,10 @@ std::optional<Style::ElementStyle> SliderThumbElement::resolveCustomStyle(const
 {
     if (!hostStyle)
         return std::nullopt;
-
-    switch (hostStyle->effectiveAppearance()) {
-    case MediaSliderPart:
-    case MediaVolumeSliderPart:
-    case MediaFullScreenVolumeSliderPart:
-        m_shadowPseudoId = ShadowPseudoIds::webkitMediaSliderThumb();
-        break;
-    default:
-        m_shadowPseudoId = ShadowPseudoIds::webkitSliderThumb();
-        break;
-    }
+    m_shadowPseudoId = ShadowPseudoIds::webkitSliderThumb();
```

Comment:
Instead of doing this, can you delete:
- https://webkit-search.igalia.com/webkit/source/Source/WebCore/html/shadow/SliderThumbElement.cpp#593-596
- https://webkit-search.igalia.com/webkit/rev/6852fe37c6f7616408223dea6a3ee7fd890a17d4/Source/WebCore/html/shadow/SliderThumbElement.h#77,96 ?

and use `setPseudo()` like here: https://webkit-search.igalia.com/webkit/rev/6852fe37c6f7616408223dea6a3ee7fd890a17d4/Source/WebCore/html/shadow/SpinButtonElement.cpp#62-67